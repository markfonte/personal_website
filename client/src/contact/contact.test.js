import React from 'react';
import Contact from './contact.jsx';
import renderer from 'react-test-renderer';

test('Contact matches snapshot', () => {
  const component = renderer.create(
      <Contact />,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
