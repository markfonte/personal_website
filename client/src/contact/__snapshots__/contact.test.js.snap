// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact matches snapshot 1`] = `
<div
  className="Contact-root-1"
>
  <h6
    className="MuiTypography-root Contact-title-2 MuiTypography-h6"
  >
    Let's get in touch!
  </h6>
  <h6
    className="MuiTypography-root Contact-subtitle-3 MuiTypography-subtitle1"
  >
    I am always open to new opportunities - recommendations, contracting work, collaborations, job opportunities, etc. Feel free to reach out!
  </h6>
  <div>
    <a
      aria-disabled={false}
      className="MuiButtonBase-root MuiButton-root MuiButton-outlined Contact-buttonDefault-5 MuiButton-outlinedSecondary"
      href="mailto:mark@fonte.com"
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
    >
      <span
        className="MuiButton-label"
      >
        Email
      </span>
    </a>
    <a
      aria-disabled={false}
      className="MuiButtonBase-root MuiButton-root MuiButton-outlined Contact-buttonDefault-5 MuiButton-outlinedSecondary"
      href="tel:406-948-1034"
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
    >
      <span
        className="MuiButton-label"
      >
        Phone
      </span>
    </a>
    <a
      aria-disabled={false}
      className="MuiButtonBase-root MuiButton-root MuiButton-outlined Contact-buttonDefault-5 MuiButton-outlinedSecondary"
      href="https://venmo.com/Mark-Fonte"
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
    >
      <span
        className="MuiButton-label"
      >
        Venmo
      </span>
    </a>
  </div>
  <span
    className="MuiTypography-root Contact-paragraph-4 MuiTypography-caption"
  >
    My resume is available upon request
  </span>
  <div>
    <div
      className="LI-profile-badge"
      data-locale="en_US"
      data-size="large"
      data-theme="dark"
      data-type="horizontal"
      data-vanity="mark-fonte"
      data-version="v1"
    >
      <a
        className="LI-simple-link"
        href="https://www.linkedin.com/in/mark-fonte?trk=profile-badge"
      >
        Mark Fonte on LinkedIn
      </a>
    </div>
  </div>
  <div
    className="Contact-paragraph-4"
  >
    <img
      alt="Temp"
      src={
        Object {
          "process": [Function],
        }
      }
      style={
        Object {
          "maxHeight": "500px",
          "maxWidth": "100%",
        }
      }
    />
  </div>
</div>
`;
